src/apps/admin/app.py

```
@@ -101,6 +104,48 @@ async def get_repository_status():
         return {"error": str(e)}
 
 
+@app.post("/api/sync")
+async def sync_stream(
+    coordinator: SyncCoordinator = Depends(get_sync_coordinator),
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)